src/components/List.js

```diff
@@ -12,11 +12,34 @@ const List = ({ token, results }) => {
             Add Item
           </NavLink>
         </>
-      ) : null}
+      ) : (
+        <>
+          <label htmlFor="searchField">Search</label>
+          <input
+            onChange={event => setSearchTerm(event.target.value)}
+            value={searchTerm}
+            id="searchField"
+            style={{ margin: 10 }}
+          ></input>
+          <button
+            disabled={searchTerm === ''}
+            onClick={() => setSearchTerm('')}
+          >
+            x
+          </button>
```

Comment:
Added this to the text field in AddItem as well! We'll have to add a class to both so we can style them consistently